tdvt/tdvt/exprtests/standard/setup.string.startswith.null.txt

```diff
@@ -0,0 +1,5 @@
+STARTSWITH(str2, 'o')
+STARTSWITH(Int0, Null)
+STARTSWITH(Num0, Null)
```

Comment:
same for the above - if I'm missing something about this let me know

nit: can remove the blank lines below, too